Updating Swift x S3 API AWS CLI configuration (#7518)
ovh/docs: pages/storage_and_backup/object_storage/pcs_getting_started_with_the_swift_s3_api/guide.en-gb.md

### Configure AWS client

> You can either follow the CLI way or create AWS configuration files manually.

Reviewer: ```suggestion
You can either use the CLI method or create AWS configuration files manually.
```

Install the AWS client and configure it as follows:

> [!tabs]
> **Python package**
>>
>> ```bash
>> user@host:~$ pip install awscli
>> ```
>> 

> > **Debian-based OS's**

Reviewer: ```suggestion
> **Debian-based OS**
```

>>
>> ```bash
>> user@host:~$ sudo apt install awscli
>> ```
>> 

> > **RHEL-based OS's**

Reviewer: ```suggestion
> **RHEL-based OS**
```

>>
>> ```bash 
>> user@host:~$ sudo yum install awscli
>> ```
>> 

> [!tabs]
> CLI method
>>
>> ```bash
>> user@host:~$ aws configure
>> ```

> >> Then, follow the steps and enter your AWS credentials generated by the commands above

Reviewer: ```suggestion
>> Then, follow the steps and enter your AWS credentials generated by the commands above.
```
